www/plugin.js

```diff
@@ -159,11 +159,12 @@ var Purchases = /** @class */ (function () {
      * Set this if you would like the RevenueCat SDK to store its preferences in a different NSUserDefaults
      * suite, otherwise it will use standardUserDefaults. Default is null, which will make the SDK use standardUserDefaults.
      */
-    Purchases.configure = function (apiKey, appUserID, observerMode, userDefaultsSuiteName) {
+    Purchases.configure = function (apiKey, appUserID, observerMode, userDefaultsSuiteName, useAmazon) {
         if (observerMode === void 0) { observerMode = false; }
+        if (useAmazon === void 0) { useAmazon = false; }
         window.cordova.exec(function (customerInfo) {
             window.cordova.fireWindowEvent("onCustomerInfoUpdated", customerInfo);
-        }, null, PLUGIN_NAME, "configure", [apiKey, appUserID, observerMode, userDefaultsSuiteName]);
+        }, null, PLUGIN_NAME, "configure", [apiKey, appUserID, observerMode, userDefaultsSuiteName || null, useAmazon]);
```

Comment:
I wonder, why do we need to add the `|| null`?

Reply:
hmm looks like I made this change because I changed this [`50a8791` (#110)](https://github.com/RevenueCat/cordova-plugin-purchases/pull/110/commits/50a8791115a065a459ace033585f75ac4a2a6612#diff-5776869548f9a7e95693a9bf21febcb3d6e3c808fc9ed4533444e03781400958L573)

But I am not sure why I did that 🤔 Leaving it with the `= null` would have the same effect right?

Reply:
Actually, it's because I made userDefaultsSuiteName not nullable in [`50a8791` (#110)](https://github.com/RevenueCat/cordova-plugin-purchases/pull/110/commits/50a8791115a065a459ace033585f75ac4a2a6612#diff-5776869548f9a7e95693a9bf21febcb3d6e3c808fc9ed4533444e03781400958R499)

Reply:
I removed this change